### DISCUSSION TITLE
Rebranding from uYouPlusExtra to uYouPlusEnhanced

### DISCUSSION THREAD

**Discussion by Infinitay** [arichornlover/uYouEnhanced#269] [Q&A]
I looked around the repo but couldn't figure out an exact reason for it. Are there any major changes? Also, will our settings be reset going from extra to enhanced?

**Comment by arichornlover** [comment]
Nope, it won’t reset. But the uYouEnhanced settings will look different next release.

https://github.com/arichornlover/uYouEnhanced/assets/78001398/b684531e-b41a-452f-a3d2-7bb99b366e52
